### ISSUE TITLE
NVDA Failing to Read Vs Code After Open for Extended Period

### ISSUE THREAD

**Issue by ironcross32** [nvaccess/nvda#9021]
### Steps to reproduce:
Work with vs code and NVDA, then do something else on the system, after alt-tabbling bak in, especially after vs code is open for a while, NVDA sometimes stops reading anything in the window, seemingly stuck in focus mode as well.
### Actual behavior:
NVDA no longer reads most things in the window, with the exception of pressing the ALt number row keys to switch window, it will still read the title of the new window.
### Expected behavior:
NVDA should read anything in the app, no matter how long it has been left open, or how many times you switch windows. As long as the app itself is responding and behaving normally, NVDA should continue to work with it.
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
NVDA 2018.3.2
#### Windows version:
Windows 10 version 1809 version 17763.134
#### Name and version of other software in use when reproducing the issue:
Microsoft Visual Studio Code Version: 1.29.1 (user setup)
Commit: bc24f98b5f70467bc689abf41cc5550ca637088e
Date: 2018-11-15T19:13:36.375Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64
#### Other information about your system:
none that applies
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by Neurrone** [comment]
This is a [known VS Code issue](https://github.com/Microsoft/vscode/issues/28316).

**Comment by ironcross32** [comment]
Thanks, and seeing as how on the vs code issue tracker, its identified as upstream, it seems that little to nothing will be done about that. Disappointing, but not unexpected. I have produced a workaround. By adding an entry to keybindings.json, I was able to create a keyboard to force a restart of vs code which then enables NVDA to regain focus.

**Comment by Neurrone** [comment]
I've implemented an add-on that works around this issue. See the [last comment in the Code tracking issue](https://github.com/Microsoft/vscode/issues/28316#issuecomment-493683016)

**Comment by Adriani90** [comment]
The issue on VS Code side has been fixed, thanks for posting the link here so that we could keep track on this. Closing as works for me.
